config.js

```diff
@@ -3,6 +3,11 @@ function load(envName) {
         ...require('./config/base.json'),
         ...loadConfig(envName),
         ...loadConfig('local'),
+        dev: {
```

Comment:
@Hinton 
Seems weird we need to load these twice to get a property.

Does the override not work properly?

Reply:
I wanted to support merging `dev` properties, as a common use case is to override `allowedHosts` while keeping the environment specific `proxy`. I suppose we could do a deep merge instead.